### ISSUE TITLE
quickstart: Funny order of "how many nodes"

### ISSUE THREAD

**Issue by pinheadmz** [bitcoin-dev-project/warnet#525]
I think "Enter the number of nodes" should be asked immediately after "How many nodes would you like?" if the response is "other"

```
[?] Do you want to create a new project? (Y/n): 
[?] Enter the project directory path: wn100
[?] Do you want to create a custom network? (Y/n): 
[?] Enter your network name: wn100
[?] How many nodes would you like?: 
   8
   12
   20
   50
 > other

[?] How many connections would you like each node to have?: 
   0
   1
   2
 > 8
   12
   other

[?] Which version would you like nodes to run by default?: 
 > 27.0
   26.0
   25.1
   24.2
   23.2
   22.2

[?] Enter the number of nodes: 100
```

**Comment by willcl-ark** [comment]
I’ll take this. I didn’t like the way this worked the other day.

**Comment by willcl-ark** [comment]
This is addressed as part of #536
